.pre-commit-config.yaml

```diff
@@ -1,9 +1,14 @@
 repos:
--   repo: https://github.com/antonbabenko/pre-commit-terraform
+  - repo: https://github.com/antonbabenko/pre-commit-terraform
     rev: v1.81.0
     hooks:
-        - id: terraform_docs
-          args:
+      - id: terraform_docs
+        args:
             - --args=--config=.terraform-docs.yml
-        - id: terraform_fmt
-        - id: terragrunt_fmt  
\ No newline at end of file
+      - id: terraform_fmt
+      - id: terragrunt_fmt
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0  # Use the ref you want to point at
+    hooks:
+      - id: trailing-whitespace
+        exclude: "^compliance/|^docs/"
```

Comment:
I think we should include `docs/` (those are written by humans after all)

There it makes the most sense and is less invasive than markdownlint etc.

Not sure about hcl and why terraform fmt doesn't enforce trailing whitespace but ok. It's a minor stylistic choice so I'd say we should just roll with it instead of discussing extensively